### PR TITLE
Update run.apls

### PR DESCRIPTION
**Pull request by pmikkelsen**

### CODE REVIEW

**Comment by pmikkelsen** [comment]
Running `dyalogscript run.apls` now correctly loads StartupSession.aplf, and links in the sources from the current directory. The logic for choosing the tests and options has been rewritten so it is less likely that invalid input causes unexpected errors.

The names of the tests are picked up dynamically by looking at which namespaces exists under #.tests.

File: run.apls [hunk 1]
```diff
@@ -2,48 +2,74 @@
 
 ⍝ Full session start-up
 DYALOG ← 2 ⎕NQ #'GetEnvironment' 'DYALOG'
```
Comment (sloorush) [inline run.apls:4]:
```suggestion
```

File: run.apls [hunk 2]
```diff
@@ -2,48 +2,74 @@
...
-    3::'Test not found, please select from the list'⎕SIGNAL 11
-    tIndex ←ts⍳⊂tName
-    tName≡'all':(⊂'tests.'),¨ts
-    'tests.',⊃ts[tIndex]
-}⍬
-
-run←{
-    tName≡'all' : unittest.RunTests ¨(⍎¨t)
-    unittest.RunTests (⍎t) verbose stop reps randLink
+⍎⊃2⎕FIX 'file://',DYALOG,'/StartupSession.aplf'
```
Comment (sloorush) [inline run.apls:5]:
```suggestion
⎕SE.(⍎⊃2⎕FIX'/StartupSession.aplf',⍨2⎕NQ#'GetEnvironment' 'DYALOG')
```

Comment (sloorush) [reply run.apls:5]: Can you check if this works? Got this from aplcart while trying to fix my dyalogscript installatio

**Review by sloorush — APPROVED** [review]

**Pull request merged**